fp4.js

```diff
@@ -0,0 +1,120 @@
+//Creating and Array
+
+const cars = ["Saab", "Volvo", "BMW"];
+document.getElementById("demo1").innerHTML = cars
+
+//Using the JavaScript Keyword new
+const rides = new Array ("Saab", "Volvo", "BMW");
+document.getElementById("demo2").innerHTML =rides
+
+//Accessing Array Elements
+document.getElementById("demo3").innerHTML = cars[0]
+document.getElementById("demo4").innerHTML =rides[2]
+
+//Changing an Array Element
+const students = ["Isaac", "Bobai", "Akutsan", "David"];
+document.getElementById("demo5").innerHTML = students
+students [3] = "Bakyung";
+document.getElementById("demo6").innerHTML = students
```

Comment:
- Kindly look at changing the array element, was "Bakyung" added? 🤔

Reply:
Yes sir, array element [3] was changed from David to Bakyung and not just adding Bakyung to the students array.